examples/wifi/espnow/main/espnow_example_main.c

```diff
@@ -321,13 +321,13 @@ static esp_err_t example_espnow_init(void)
 
     /* Initialize sending parameters. */
     send_param = malloc(sizeof(example_espnow_send_param_t));
-    memset(send_param, 0, sizeof(example_espnow_send_param_t));
```

Comment:
Alright, so the reason to use calloc is to avoid the redundancy. calloc = malloc + memset.